common/src/main/scala/org/apache/comet/CometConf.scala

```diff
@@ -84,15 +84,33 @@ object CometConf extends ShimCometConf {
     .booleanConf
     .createWithDefault(sys.env.getOrElse("ENABLE_COMET", "true").toBoolean)
 
-  val COMET_SCAN_ENABLED: ConfigEntry[Boolean] = conf("spark.comet.scan.enabled")
+  val COMET_NATIVE_SCAN_ENABLED: ConfigEntry[Boolean] = conf("spark.comet.scan.enabled")
     .doc(
-      "Whether to enable Comet scan. When this is turned on, Spark will use Comet to read " +
-        "Parquet data source. Note that to enable native vectorized execution, both this " +
-        "config and 'spark.comet.exec.enabled' need to be enabled. By default, this config " +
-        "is true.")
+      "Whether to enable native scans. When this is turned on, Spark will use Comet to " +
+        "read supported data sources (currently only Parquet is supported natively). Note " +
+        "that to enable native vectorized execution, both this config and " +
+        "'spark.comet.exec.enabled' need to be enabled. By default, this config is true.")
     .booleanConf
     .createWithDefault(true)
 
+  val COMET_CONVERT_FROM_PARQUET_ENABLED: ConfigEntry[Boolean] =
+    conf("spark.comet.convert.parquet.enabled")
+      .doc(
+        "When enabled, data from Parquet v1 and v2 scans will be converted to Arrow format. Note " +
+          "that to enable native vectorized execution, both this config and " +
+          "'spark.comet.exec.enabled' need to be enabled.")
+      .booleanConf
+      .createWithDefault(false)
```

Comment:
This might be confusing to `COMET_NATIVE_SCAN_ENABLED`. We probably can mention in the doc that this doesn't use Comet native scan but Spark parquet reader and convert it to Comet.

Reply:
I added this in the docs:

```
## Parquet

 When `spark.comet.scan.enabled` is enabled, Parquet scans will be performed natively by Comet if all data types
 in the schema are supported. When this option is not enabled, the scan will fall back to Spark. In this case,
 enabling `spark.comet.convert.parquet.enabled` will immediately convert the data into Arrow format, allowing native 
 execution to happen after that, but the process may not be efficient.
 ```
 
 I'll take another pass at the config description though to make it more detailed